<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Sekura::BottomTableWidget</class>
 <widget class="QWidget" name="Sekura::BottomTableWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>47</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QPushButton" name="pbCreate">
     <property name="text">
      <string>Create</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/images/029-floppy-disk.png</normaloff>:/images/029-floppy-disk.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pbDrop">
     <property name="text">
      <string>Drop</string>
     </property>
     <property name="icon">
      <iconset resource="resources.qrc">
       <normaloff>:/images/030-dot.png</normaloff>:/images/030-dot.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>175</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
